dotnet/src/SemanticKernel.Core/PromptTemplate/NoopPromptTemplate.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.SemanticKernel;
+
+/// <summary>
+/// No operation implementation of <see cref="IPromptTemplate"/> that just returns the prompt template.
+/// </summary>
+internal sealed class NoopPromptTemplate : IPromptTemplate
```

Comment:
Consider adding UT or an `ExcludeCodeCoverage` for this.

```suggestion
internal sealed class NoopPromptTemplate : IPromptTemplate
```